### PR TITLE
chore (docs): change naming to AI SDK

### PR DESCRIPTION
**Pull request by lgrammel**

### CODE REVIEW

File: content/docs/02-getting-started/03-nextjs-pages-router.mdx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Next.js Pages Router
-description: Welcome to the Vercel AI SDK documentation!
+description: Welcome to the AI SDK documentation!
 ---
 
 # Next.js Pages Router Quickstart
 
-The Vercel AI SDK is a powerful Typescript library designed to help developers build AI-powered applications.
+the AI SDK is a powerful Typescript library designed to help developers build AI-powered applications.
```
Comment (nicoalbanese) [inline content/docs/02-getting-started/03-nextjs-pages-router.mdx:8]: capitalise

File: content/docs/02-getting-started/04-svelte.mdx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Svelte
-description: Welcome to the Vercel AI SDK quickstart guide for Svelte!
+description: Welcome to the AI SDK quickstart guide for Svelte!
 ---
 
 # Svelte Quickstart
 
-The Vercel AI SDK is a powerful Typescript library designed to help developers build AI-powered applications.
+the AI SDK is a powerful Typescript library designed to help developers build AI-powered applications.
```
Comment (nicoalbanese) [inline content/docs/02-getting-started/04-svelte.mdx:8]: capitalise

File: content/docs/02-getting-started/05-nuxt.mdx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Nuxt
-description: Welcome to the Vercel AI SDK documentation!
+description: Welcome to the AI SDK documentation!
 ---
 
 # Nuxt Quickstart
 
-The Vercel AI SDK is a powerful Typescript library designed to help developers build AI-powered applications.
+the AI SDK is a powerful Typescript library designed to help developers build AI-powered applications.
```
Comment (nicoalbanese) [inline content/docs/02-getting-started/05-nuxt.mdx:8]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-description: Get started with Llama 3.1 using the Vercel AI SDK.
+description: Get started with Llama 3.1 using the AI SDK.
 ---
 
 # Get started with Llama 3.1
 
 With the [release of Llama 3.1](https://ai.meta.com/blog/meta-llama-3-1/), there has never been a better time to start building AI applications.
 
-The [Vercel AI SDK](/) is a powerful TypeScript toolkit for building AI application with large language models (LLMs) like Llama 3.1 alongside popular frameworks like React, Next.js, Vue, Svelte, Node.js, and more
+the [AI SDK](/) is a powerful TypeScript toolkit for building AI application with large language models (LLMs) like Llama 3.1 alongside popular frameworks like React, Next.js, Vue, Svelte, Node.js, and more
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:10]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 5]
```diff
@@ -26,13 +26,13 @@ Llama 3.1 includes a new 405B parameter model, becoming the largest open-source
 
 When choosing between the different sizes of Llama 3.1 models (405B, 70B, 8B), consider the trade-off between performance and computational requirements. The 405B model offers the highest accuracy and capability for complex tasks but requires significant computational resources. The 70B model provides a good balance of performance and efficiency for most applications, while the 8B model is suitable for simpler tasks or resource-constrained environments where speed and lower computational overhead are priorities.
 
-## Getting Started with the Vercel AI SDK
+## Getting Started with the AI SDK
 
-The Vercel AI SDK is the TypeScript toolkit designed to help developers build AI-powered applications with React, Next.js, Vue, Svelte, Node.js, and more. Integrating LLMs into applications is complicated and heavily dependent on the specific model provider you use.
+the AI SDK is the TypeScript toolkit designed to help developers build AI-powered applications with React, Next.js, Vue, Svelte, Node.js, and more. Integrating LLMs into applications is complicated and heavily dependent on the specific model provider you use.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:31]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 6]
```diff
@@ -26,13 +26,13 @@ Llama 3.1 includes a new 405B parameter model, becoming the largest open-source
...
 
-The Vercel AI SDK abstracts away the differences between model providers, eliminates boilerplate code for building chatbots, and allows you to go beyond text output to generate rich, interactive components.
+the AI SDK abstracts away the differences between model providers, eliminates boilerplate code for building chatbots, and allows you to go beyond text output to generate rich, interactive components.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:33]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 7]
```diff
@@ -121,11 +121,11 @@ This code snippet will generate a type-safe recipe that conforms to the specifie
 
 While LLMs have incredible generation capabilities, they struggle with discrete tasks (e.g. mathematics) and interacting with the outside world (e.g. getting the weather). The solution: tools, which are like programs that you provide to the model, which it can choose to call as necessary.
 
-### Using Tools with the Vercel AI SDK
+### Using Tools with the AI SDK
 
-The Vercel AI SDK supports tool usage across several of its functions, including [`generateText`](/docs/reference/ai-sdk-core/generate-text) and [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui). By passing one or more tools to the `tools` parameter, you can extend the capabilities of LLMs, allowing them to perform discrete tasks and interact with external systems.
+the AI SDK supports tool usage across several of its functions, including [`generateText`](/docs/reference/ai-sdk-core/generate-text) and [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui). By passing one or more tools to the `tools` parameter, you can extend the capabilities of LLMs, allowing them to perform discrete tasks and interact with external systems.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:126]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 8]
```diff
@@ -163,9 +163,9 @@ Agents take your AI applications a step further by allowing models to execute mu
 
 Agents use LLMs to choose the next step in a problem-solving process. They can reason at each step and make decisions based on the evolving context.
 
-### Implementing Agents with the Vercel AI SDK
+### Implementing Agents with the AI SDK
 
-The Vercel AI SDK supports agent implementation through the `maxSteps` parameter. This allows the model to make multiple decisions and tool calls in a single interaction.
+the AI SDK supports agent implementation through the `maxSteps` parameter. This allows the model to make multiple decisions and tool calls in a single interaction.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:168]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 9]
```diff
@@ -267,7 +267,7 @@ This enables a seamless chat experience where the user can see the AI response a
 
 ### Going Beyond Text
 
-The Vercel AI SDK's React Server Components (RSC) API enables you to create rich, interactive interfaces that go beyond simple text generation. With the [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui) function, you can dynamically stream React components from the server to the client.
+the AI SDK's React Server Components (RSC) API enables you to create rich, interactive interfaces that go beyond simple text generation. With the [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui) function, you can dynamically stream React components from the server to the client.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:270]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 10]
```diff
@@ -412,13 +412,13 @@ To learn more about fine-tuning Llama models, check out these resources:
 
 ## Conclusion
 
-The Vercel AI SDK offers a powerful and flexible way to integrate cutting-edge AI models like Llama 3.1 into your applications. With AI SDK Core, you can seamlessly switch between different AI models and providers by changing just two lines of code. This flexibility allows for quick experimentation and adaptation, reducing the time required to change models from days to minutes.
+the AI SDK offers a powerful and flexible way to integrate cutting-edge AI models like Llama 3.1 into your applications. With AI SDK Core, you can seamlessly switch between different AI models and providers by changing just two lines of code. This flexibility allows for quick experimentation and adaptation, reducing the time required to change models from days to minutes.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:415]: capitalise

File: content/docs/02-guides/03-llama-3_1.mdx [hunk 11]
```diff
@@ -412,13 +412,13 @@ To learn more about fine-tuning Llama models, check out these resources:
...
 
-The Vercel AI SDK ensures that your application remains clean and modular, accelerating development and future-proofing against the rapidly evolving landscape.
+the AI SDK ensures that your application remains clean and modular, accelerating development and future-proofing against the rapidly evolving landscape.
```
Comment (nicoalbanese) [inline content/docs/02-guides/03-llama-3_1.mdx:417]: capitalise

File: content/docs/02-guides/04-o1.mdx [hunk 12]
```diff
@@ -1,13 +1,13 @@
...
-description: Get started with OpenAI o1 using the Vercel AI SDK.
+description: Get started with OpenAI o1 using the AI SDK.
 ---
 
 # Get started with OpenAI o1
 
 With the [release of OpenAI's o1 series models](https://openai.com/index/introducing-openai-o1-preview/), there has never been a better time to start building AI applications, particularly those that require complex reasoning capabilities.
 
-The [Vercel AI SDK](/) is a powerful TypeScript toolkit for building AI applications with large language models (LLMs) like OpenAI o1 alongside popular frameworks like React, Next.js, Vue, Svelte, Node.js, and more.
+the [AI SDK](/) is a powerful TypeScript toolkit for building AI applications with large language models (LLMs) like OpenAI o1 alongside popular frameworks like React, Next.js, Vue, Svelte, Node.js, and more.
```
Comment (nicoalbanese) [inline content/docs/02-guides/04-o1.mdx:10]: capitalise

File: content/docs/02-guides/04-o1.mdx [hunk 13]
```diff
@@ -45,13 +45,13 @@ The o1 models perform best with straightforward prompts. Some prompt engineering
 3. Use delimiters for clarity: Use delimiters like triple quotation marks, XML tags, or section titles to clearly indicate distinct parts of the input, helping the model interpret different sections appropriately.
 4. Limit additional context in retrieval-augmented generation (RAG): When providing additional context or documents, include only the most relevant information to prevent the model from overcomplicating its response.
 
-## Getting Started with the Vercel AI SDK
+## Getting Started with the AI SDK
 
-The Vercel AI SDK is the TypeScript toolkit designed to help developers build AI-powered applications with React, Next.js, Vue, Svelte, Node.js, and more. Integrating LLMs into applications is complicated and heavily dependent on the specific model provider you use.
+the AI SDK is the TypeScript toolkit designed to help developers build AI-powered applications with React, Next.js, Vue, Svelte, Node.js, and more. Integrating LLMs into applications is complicated and heavily dependent on the specific model provider you use.
```
Comment (nicoalbanese) [inline content/docs/02-guides/04-o1.mdx:50]: capitalise

File: content/docs/02-guides/04-o1.mdx [hunk 14]
```diff
@@ -45,13 +45,13 @@ The o1 models perform best with straightforward prompts. Some prompt engineering
...
 
-The Vercel AI SDK abstracts away the differences between model providers, eliminates boilerplate code for building chatbots, and allows you to go beyond text output to generate rich, interactive components.
+the AI SDK abstracts away the differences between model providers, eliminates boilerplate code for building chatbots, and allows you to go beyond text output to generate rich, interactive components.
```
Comment (nicoalbanese) [inline content/docs/02-guides/04-o1.mdx:52]: capitalise

File: content/docs/03-ai-sdk-core/05-generating-text.mdx [hunk 15]
```diff
@@ -1,14 +1,14 @@
...
+description: Learn how to generate text with the AI SDK.
 ---
 
 # Generating and Streaming Text
 
 Large language models (LLMs) can generate text in response to a prompt, which can contain instructions and information to process.
 For example, you can ask a model to come up with a recipe, draft an email, or summarize a document.
 
-The Vercel AI SDK Core provides two functions to generate text and stream it from LLMs:
+the AI SDK Core provides two functions to generate text and stream it from LLMs:
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/05-generating-text.mdx:11]: I would maybe remove the "the" here

File: content/docs/03-ai-sdk-core/10-generating-structured-data.mdx [hunk 16]
```diff
@@ -11,7 +11,7 @@ For example, you might want to extract information from text, classify data, or
 Many language models are capable of generating structured data, often defined as using "JSON modes" or "tools".
 However, you need to manually provide schemas and then validate the generated data as LLMs can produce incorrect or incomplete structured data.
 
-The Vercel AI SDK standardises structured object generation across model providers
+the AI SDK standardises structured object generation across model providers
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/10-generating-structured-data.mdx:14]: capitalise

File: content/docs/03-ai-sdk-core/30-embeddings.mdx [hunk 17]
```diff
@@ -10,7 +10,7 @@ In this space, similar words are close to each other, and the distance between w
 
 ## Embedding a Single Value
 
-The Vercel AI SDK provides the [`embed`](/docs/reference/ai-sdk-core/embed) function to embed single values, which is useful for tasks such as finding similar words
+the AI SDK provides the [`embed`](/docs/reference/ai-sdk-core/embed) function to embed single values, which is useful for tasks such as finding similar words
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/30-embeddings.mdx:13]: capitalise

File: content/docs/03-ai-sdk-core/30-embeddings.mdx [hunk 18]
```diff
@@ -30,7 +30,7 @@ const { embedding } = await embed({
 When loading data, e.g. when preparing a data store for retrieval-augmented generation (RAG),
 it is often useful to embed many values at once (batch embedding).
 
-The Vercel AI SDK provides the [`embedMany`](/docs/reference/ai-sdk-core/embed-many) function for this purpose.
+the AI SDK provides the [`embedMany`](/docs/reference/ai-sdk-core/embed-many) function for this purpose.
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/30-embeddings.mdx:33]: capitalise

File: content/docs/03-ai-sdk-core/40-provider-management.mdx [hunk 19]
```diff
@@ -10,7 +10,7 @@ description: Learn how to work with multiple providers
 When you work with multiple providers and models, it is often desirable to manage them in a central place
 and access the models through simple string ids.
 
-The Vercel AI SDK offers [custom providers](/docs/reference/ai-sdk-core/custom-provider) and
+the AI SDK offers [custom providers](/docs/reference/ai-sdk-core/custom-provider) and
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/40-provider-management.mdx:13]: capitalise

File: content/docs/02-getting-started/02-nextjs-app-router.mdx [hunk 20]
```diff
@@ -249,9 +249,9 @@ Head back to your browser (http://localhost:3000) and enter a new message. You s
 
 ## Introducing more granular control with `ai/rsc`
 
-So far, you have used Vercel AI SDK's [ UI ](/docs/ai-sdk-ui) package to connect your frontend to your API route. This package is framework agnostic and provides simple abstractions for quickly building chat-like interfaces with LLMs.
+So far, you have used the AI SDK's [ UI ](/docs/ai-sdk-ui) package to connect your frontend to your API route. This package is framework agnostic and provides simple abstractions for quickly building chat-like interfaces with LLMs.
 
-The Vercel AI SDK also has a package ([`ai/rsc`](/docs/ai-sdk-rsc)) specifically designed for frameworks that support the [React Server Component](https://nextjs.org/docs/app/building-your-application/rendering/server-components) architecture. With `ai/rsc`, you can build AI applications that go beyond pure text.
+the AI SDK also has a package ([`ai/rsc`](/docs/ai-sdk-rsc)) specifically designed for frameworks that support the [React Server Component](https://nextjs.org/docs/app/building-your-application/rendering/server-components) architecture. With `ai/rsc`, you can build AI applications that go beyond pure text.
```
Comment (lgrammel) [inline content/docs/02-getting-started/02-nextjs-app-router.mdx:254]: f

File: content/docs/03-ai-sdk-core/60-telemetry.mdx [hunk 21]
```diff
@@ -5,7 +5,7 @@ description: Using OpenTelemetry with AI SDK Core
 
 # Telemetry
 
-The Vercel AI SDK uses [OpenTelemetry](https://opentelemetry.io/) to collect telemetry data.
+the AI SDK uses [OpenTelemetry](https://opentelemetry.io/) to collect telemetry data.
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/60-telemetry.mdx:8]: capitalise

File: content/docs/03-ai-sdk-core/index.mdx [hunk 22]
```diff
@@ -10,7 +10,7 @@ description: Learn about the Vercel AI SDK Core.
     {
       title: 'Overview',
       description:
-        'Learn about the Vercel AI SDK Core and how to work with Large Language Models (LLMs).',
+        'Learn about the AI SDK Core and how to work with Large Language Models (LLMs).',
```
Comment (nicoalbanese) [inline content/docs/03-ai-sdk-core/index.mdx:13]: for all of these I would remove the "the"

File: content/docs/04-ai-sdk-rsc/index.mdx [hunk 23]
```diff
@@ -1,6 +1,6 @@
 ---
 title: AI SDK RSC
-description: Learn about the Vercel AI SDK RSC.
+description: Learn about the AI SDK RSC.
```
Comment (nicoalbanese) [inline content/docs/04-ai-sdk-rsc/index.mdx:3]: and here

File: content/docs/05-ai-sdk-ui/03-chatbot-with-tool-calling.mdx [hunk 24]
```diff
@@ -6,7 +6,7 @@ description: Learn how to use tools with the useChat hook.
 # Chatbot with Tools
 
 With `useChat` and `streamText`, you can use tools in your chatbot application.
-The Vercel AI SDK supports three types of tools in this context:
+the AI SDK supports three types of tools in this context:
```
Comment (nicoalbanese) [inline content/docs/05-ai-sdk-ui/03-chatbot-with-tool-calling.mdx:9]: capitalise

File: content/docs/05-ai-sdk-ui/15-storing-messages.mdx [hunk 25]
```diff
@@ -1,12 +1,12 @@
...
 title: Storing Messages
-description: Welcome to the Vercel AI SDK documentation!
+description: Welcome to the AI SDK documentation!
 ---
 
 # Storing Messages
 
 The ability to store message history is essential for chatbot use cases.
-The Vercel AI SDK simplifies the process of storing chat history through the `onFinish` callback of the `streamText` function.
+the AI SDK simplifies the process of storing chat history through the `onFinish` callback of the `streamText` function.
```
Comment (nicoalbanese) [inline content/docs/05-ai-sdk-ui/15-storing-messages.mdx:9]: capitalise

File: content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-3-1.mdx [hunk 26]
```diff
@@ -1,29 +1,29 @@
 ---
 title: Migrate 3.0 to 3.1
-description: Learn how to migrate from Vercel AI SDK 3.0 to 3.1.
+description: Learn how to migrate from the AI SDK 3.0 to 3.1.
```
Comment (nicoalbanese) [inline content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-3-1.mdx:3]: remove "the"

File: content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-3-1.mdx [hunk 27]
```diff
@@ -1,29 +1,29 @@
...
 ---
 
 # Migration Guide (3.0 to 3.1)
 
 This guide will help you:
 
-- Upgrade to Vercel AI SDK 3.1
-- Migrate from [Legacy Providers](/providers/legacy-providers) to Vercel AI SDK Core
+- Upgrade to the AI SDK 3.1
```
Comment (nicoalbanese) [inline content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-3-1.mdx:10]: remove "the"

File: content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-3-1.mdx [hunk 28]
```diff
@@ -1,29 +1,29 @@
...
+- Migrate from [Legacy Providers](/providers/legacy-providers) to AI SDK Core
 - Migrate from [`render`](/docs/reference/ai-sdk-rsc/render) to [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui)
 
-Upgrading to Vercel AI SDK 3.1 does not require using the newly released AI SDK Core API or [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui) function. You can continue to use Vercel AI SDK with [existing (Legacy) providers](/providers/legacy-providers).
+Upgrading to the AI SDK 3.1 does not require using the newly released AI SDK Core API or [`streamUI`](/docs/reference/ai-sdk-rsc/stream-ui) function. You can continue to use the AI SDK with [existing (Legacy) providers](/providers/legacy-providers).
```
Comment (nicoalbanese) [inline content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-3-1.mdx:14]: remove "the"

File: content/docs/08-troubleshooting/01-migration-guide/02-migration-guide-3-2.mdx [hunk 29]
```diff
@@ -1,21 +1,21 @@
 ---
 title: Migrate 3.1 to 3.2
-description: Learn how to migrate from Vercel AI SDK 3.1 to 3.2.
+description: Learn how to migrate from the AI SDK 3.1 to 3.2.
```
Comment (nicoalbanese) [inline content/docs/08-troubleshooting/01-migration-guide/02-migration-guide-3-2.mdx:3]: remove "the"

File: content/docs/08-troubleshooting/01-migration-guide/02-migration-guide-3-2.mdx [hunk 30]
```diff
@@ -1,21 +1,21 @@
...
 ---
 
 # Migration Guide (3.1 to 3.2)
 
-This guide will help you upgrade to Vercel AI SDK 3.2:
+This guide will help you upgrade to the AI SDK 3.2:
```
Comment (nicoalbanese) [inline content/docs/08-troubleshooting/01-migration-guide/02-migration-guide-3-2.mdx:8]: remove "the"

File: content/docs/08-troubleshooting/01-migration-guide/03-migration-guide-3-3.mdx [hunk 31]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Migrate 3.2 to 3.3
-description: Learn how to migrate from Vercel AI SDK 3.3 to 3.3.
+description: Learn how to migrate from AI SDK 3.3 to 3.3.
```
Comment (nicoalbanese) [inline content/docs/08-troubleshooting/01-migration-guide/03-migration-guide-3-3.mdx:3]: update to be from 3.2 to 3.3

File: content/examples/01-next-app/index.mdx [hunk 32]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Next.js App Router
-description: Explore examples of how to use Vercel AI SDK in Next.js App Router Application.
+description: Explore examples of how to use the AI SDK in Next.js App Router Application.
 ---
 
 # Examples for Next.js App Router
 
-The Vercel AI SDK is designed to work with Next.js App Router. Since the App Router has first class support for React Server Components, you can use the `ai/rsc` module to build applications with features like [Generative User Interfaces](/docs/ai-sdk-rsc) and better [Server-Client State Management](/docs/ai-sdk-rsc/generative-ui-state).
+the AI SDK is designed to work with Next.js App Router. Since the App Router has first class support for React Server Components, you can use the `ai/rsc` module to build applications with features like [Generative User Interfaces](/docs/ai-sdk-rsc) and better [Server-Client State Management](/docs/ai-sdk-rsc/generative-ui-state).
```
Comment (nicoalbanese) [inline content/examples/01-next-app/index.mdx:8]: capitalise

File: content/examples/index.mdx [hunk 33]
```diff
@@ -9,8 +9,8 @@ description: Explore examples of how to use Vercel AI SDK in different framework
 
 # Explore Examples of AI Applications Across Frameworks
 
-The Vercel AI SDK is designed to work with different frameworks and environments that run using Typescript or Javascript.
+the AI SDK is designed to work with different frameworks and environments that run using Typescript or Javascript.
```
Comment (nicoalbanese) [inline content/examples/index.mdx:12]: capitalise

File: content/providers/03-community-providers/01-custom-providers.mdx [hunk 34]
```diff
@@ -1,12 +1,12 @@
...
 title: Writing a Custom Provider
-description: Learn how to write a custom provider for the Vercel AI SDK
+description: Learn how to write a custom provider for the AI SDK
 ---
 
 # Writing a Custom Provider
 
-The Vercel AI SDK provides a [Language Model Specification](https://github.com/vercel/ai/tree/main/packages/provider/src/language-model/v1).
-You can write your own provider that adheres to the specification and it will be compatible with the Vercel AI SDK.
+the AI SDK provides a [Language Model Specification](https://github.com/vercel/ai/tree/main/packages/provider/src/language-model/v1).
```
Comment (nicoalbanese) [inline content/providers/03-community-providers/01-custom-providers.mdx:8]: capitalise

File: content/providers/03-community-providers/index.mdx [hunk 35]
```diff
@@ -5,7 +5,7 @@ description: Learn how to use Language Model Specification.
 
 # Community Providers
 
-The Vercel AI SDK provides a [Language Model Specification](https://github.com/vercel/ai/tree/main/packages/provider/src/language-model/v1).
+the AI SDK provides a [Language Model Specification](https://github.com/vercel/ai/tree/main/packages/provider/src/language-model/v1).
```
Comment (nicoalbanese) [inline content/providers/03-community-providers/index.mdx:8]: capitalise

File: content/providers/05-legacy-providers/openai-functions.mdx [hunk 36]
```diff
@@ -11,8 +11,8 @@ description: Learn how to use OpenAI functions with the Vercel AI SDK.
   Provider](../ai-sdk-providers/openai) instead.
 </Note>
 
-The Vercel AI SDK has **experimental support** for [OpenAI functions](https://openai.com/blog/function-calling-and-other-api-updates).
-Any of the content below is subject to change as OpenAI continues to develop their functions API and we iterate on the Vercel AI SDK. You can see our planned roadmap [here](https://twitter.com/jaredpalmer/status/1673350963191508993).
+the AI SDK has **experimental support** for [OpenAI functions](https://openai.com/blog/function-calling-and-other-api-updates).
```
Comment (nicoalbanese) [inline content/providers/05-legacy-providers/openai-functions.mdx:14]: capitalise

**Review by nicoalbanese — APPROVED** [review]

**Pull request merged**